types/jquery.appear/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for JQuery Appear 1.0
+// Project: https://github.com/bas2k/jquery.appear
+// Definitions by: Anderson Friaça <https://github.com/AndersonFriaca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+declare namespace JQueryAppear {
+    interface Options {
+        /**
+         * Data to pass into callback
+         */
+        data?: any;
+
+        /**
+         * Callback is called only in first appear
+         */
+        one?: boolean;
+
+        /**
+         * X accuracy
+         */
+        accX?: number;
+
+        /**
+         * Y accuracy
+         */
+        accY?: number;
+    }
+}
+interface JQuery {
+    /**
+     * Initialize appear plugin
+     */
+    appear(callback: ((element?: HTMLElement, data?: any) => void), options?: JQueryAppear.Options): JQuery;
```

Comment:
Should these callback parameters really be optional? See http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html .
Also, could this be `appear<T>` with `data: T` and `Options<T>` with `data: T` instead of using `any` in both places?